File: Go/SQLBoiler_ORM/dal/entities/employee.go
package entities

import "time"

type Employee struct {
	EmployeeID      int        `db:"EmployeeID"`
	LastName        string     `db:"LastName"`
	FirstName       string     `db:"FirstName"`
	Title           *string    `db:"Title"`
	TitleOfCourtesy *string    `db:"TitleOfCourtesy"`
	BirthDate       *time.Time `db:"BirthDate"`
	HireDate        *time.Time `db:"HireDate"`
	Address         *string    `db:"Address"`
	City            *string    `db:"City"`
	Region          *string    `db:"Region"`
	PostalCode      *string    `db:"PostalCode"`
	Country         *string    `db:"Country"`
	HomePhone       *string    `db:"HomePhone"`
	Extension       *string    `db:"Extension"`
	Photo           *[]byte    `db:"Photo"`
	Notes           *string    `db:"Notes"`
	ReportsTo       *int       `db:"ReportsTo"`
	PhotoPath       *string    `db:"PhotoPath"`
}

File: C#/EntityFramework_ORM/DAL/Entities/Employee.cs
﻿using DAL.Interfaces;

namespace DAL.Entities;

public partial class Employee : IEntity
{
    public int EmployeeID { get; set; }

    public string LastName { get; set; } = null!;

    public string FirstName { get; set; } = null!;

    public string? Title { get; set; }

    public string? TitleOfCourtesy { get; set; }

    public DateTime? BirthDate { get; set; }

    public DateTime? HireDate { get; set; }

    public string? Address { get; set; }

    public string? City { get; set; }

    public string? Region { get; set; }

    public string? PostalCode { get; set; }

    public string? Country { get; set; }

    public string? HomePhone { get; set; }

    public string? Extension { get; set; }

    public byte[]? Photo { get; set; }

    public string? Notes { get; set; }

    public int? ReportsTo { get; set; }

    public string? PhotoPath { get; set; }

    public virtual ICollection<EmployeeTerritories> EmployeeTerritories { get; set; } = new List<EmployeeTerritories>();

    public virtual ICollection<Employee> InverseReportsToNavigation { get; set; } = new List<Employee>();

    public virtual ICollection<Order> Orders { get; set; } = new List<Order>();

    public virtual Employee? ReportsToNavigation { get; set; }

    //public virtual ICollection<Territory> Territories { get; set; } = new List<Territory>();
}
